NethServer/Firewall.pm

```diff
@@ -951,9 +983,10 @@ sub countReferences($$)
 	'vlan' => 'role',
 	'alias' => 'role',
 	'bond' => 'role',
-        'cidr', => 'cidr',
-        'time', => 'time',
-        'iprange' => 'iprange'
+    'cidr', => 'cidr',
```

Comment:
is a github visualization issue? or che code is still not aligned?

Reply:
It was a mix between tabs and spaces: fixed.